hadoop/hadoop-quiz.md

```diff
@@ -372,7 +372,7 @@
 ### Q53. Which feature is used to roll back a corrupted HDFS instance to a previously known good point in time?
 
 - [ ] partitioning
-- [ ] snapshot
+- [x] snapshot (https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html#:~:text=is%20not%20supported.-,Snapshots,known%20good%20point%20in%20time.)
```

Comment:
```suggestion
- [x] snapshot [Reference](https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html#:~:text=is%20not%20supported.-,Snapshots,known%20good%20point%20in%20time.)
```
you can improve user visibility by using this pattern from markdown e.g. [Visible text](hidden hyperlink here www)

https://anvilproject.org/guides/content/creating-links

Reply:
@Ebazhanov Thanks. Completely agree